src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -26,12 +26,14 @@ public class JsonAdaptedPersonTest {
     private static final String INVALID_TAG = "#friend";
     private static final String INVALID_ATTENDANCE = "202-01-01";
     private static final String INVALID_BIRTHDAY = "24th of April";
+    private static final String INVALID_MATRICULATIONYEAR = "3000";
```

Comment:
Is there a reason to hard code this?

Would it make sense to make it dynamic by using something like:
`int year = Year.now().getValue();`
`INVALID_MATRICULATIONYEAR = year + 1`

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.